Converter/include/sampler_poisson.h

```diff
@@ -260,19 +260,18 @@ struct SamplerPoisson : public Sampler {
 					if (isAccepted) {
 						accepted->write(child->points->data_u8 + pointOffset, attributes.bytes);
 						// rejected->write(child->points->data_u8 + pointOffset, attributes.bytes);
-					} else {
+					} else if(numRejected > 0){
 						rejected->write(child->points->data_u8 + pointOffset, attributes.bytes);
 					}
 				}
 
 				if (numRejected == 0 && child->isLeaf()) {
 					node->children[childIndex] = nullptr;
-				} if (numRejected > 0) {
-					child->points = rejected;
-					child->numPoints = numRejected;
-
+				} 
+				child->points = rejected;

```

Comment:
Danger: if the above if block has been executed then node->children[childIndex] = nullptr; also means that child is now null too. These two lines must be placed in an else block.